pam_authz/docker/policy/display.rego

```diff
@@ -0,0 +1,20 @@
+# The package path. This should be passed with the display_endpoint flag
+# in the PAM configuration file.
+package common.display
+
+display_spec = [
+	{
+		"message": "Welcome to the OPA-PAM demonstration.",
+		"style": "info",
+	},
+	{
+		"message": "Please enter your last name: ",
```

Comment:
I would prefer "username" over "last name" in all occurrences because a person's last name is generally not what is used for authn or authz

Reply:
Wanted to use a non-username value for the example, because the username is retrieved by default from the PAM session.

Reply:
Cool makes sense, carry on :)